tests/psimulate/data/artifactless_perf_df.csv

```diff
@@ -0,0 +1,13 @@
+host,job_number,task_number,draw,seed,scenario_parameter_one,scenario_run_configuration_run_key_parameter_one,event_start,event_simulant_initialization_start,event_simulation_start,event_results_start,event_end,exec_time_setup_minutes,exec_time_simulant_initialization_minutes,exec_time_main_loop_minutes,exec_time_step_mean_seconds,exec_time_results_minutes,exec_time_total_minutes,counters_cpu_ctx_switches,counters_cpu_interrupts,counters_cpu_soft_interrupts,counters_cpu_syscalls,counters_freq_current,counters_freq_min,counters_freq_max,counters_disk_read_count,counters_disk_write_count,counters_disk_read_bytes,counters_disk_write_bytes,counters_disk_read_time,counters_disk_write_time,counters_disk_read_merged_count,counters_disk_write_merged_count,counters_disk_busy_time,counters_net_bytes_sent,counters_net_bytes_recv,counters_net_packets_sent,counters_net_packets_recv,counters_net_errin,counters_net_errout,counters_net_dropin,counters_net_dropout,counters_time
```

Comment:
I copied these over from actual summary logs and kept most of the data values. Would it be cleaner to just make these 0s or NaNs (starting at event_start)?